manifests/g/Git/Git/2.40.0/Git.Git.installer.yaml

```diff
@@ -0,0 +1,38 @@
+# Created with YamlCreate.ps1 v2.2.0 using InputObject 🤖 $debug=QUSU.CRLF..7-2-10
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Git.Git
+PackageVersion: 2.40.0
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: inno
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Commands:
+- git
+ReleaseDate: 2023-03-14
+InstallerSwitches:
+  Custom: /SUPPRESSMSGBOXES
+Installers:
+- Architecture: x86
+  Scope: machine
+  InstallerUrl: https://github.com/git-for-windows/git/releases/download/v2.40.0.windows.1/Git-2.40.0-32-bit.exe
+  InstallerSha256: 9B14E05C3EA00C51DC38838DB23FCCC6CCB21BFD42ED078BF406857FB47688D6
+- Architecture: x86
+  Scope: user
+  InstallerUrl: https://github.com/git-for-windows/git/releases/download/v2.40.0.windows.1/Git-2.40.0-32-bit.exe
+  InstallerSha256: 9B14E05C3EA00C51DC38838DB23FCCC6CCB21BFD42ED078BF406857FB47688D6
```

Comment:
```suggestion
```

Reply:
Removing these, as they'll be depreciated soon, and there doesn't seem to be a way to avoid the message box informing this.